modules/internal/ChapelLocale.chpl

```diff
@@ -474,6 +475,19 @@ module ChapelLocale {
     }
   }
 
+  // Warm up runtime components. For tasking layers that have a fixed number of
+  // threads, we create a task on each thread to warm it up (e.g. grab some
+  // initial call stacks.) We also warm up the memory layer, since allocators
+  // tend to have per thread pools/arenas.
+  private proc warmupRuntime() {
```

Comment:
I am wondering whether it makes sense to inline this. Nothing that you should spend significant time testing etc, though.

Reply:
Nah, I wouldn't expect there to be any impact from inlining it.

Reply:
(Function call overhead is nothing compared to the warmup being done or even task creation post-warmup)